ddtrace/contrib/unittest/patch.py

```diff
@@ -561,6 +600,8 @@ def _start_test_span(instance, test_suite_span: ddtrace.Span) -> ddtrace.Span:
     Starts a test  span and sets the required tags for a `unittest` test instance.
     """
     tracer = getattr(unittest, "_datadog_tracer", _CIVisibility._instance.tracer)
+    source_file_path = _get_source_file_path_for_test_method(instance)
+    start_line, end_line = _get_source_lines_for_test_method(instance)
```

Comment:
Might be worth being a little more paranoid about these possibly raising errors.